settings.js

```diff
@@ -28,7 +28,7 @@ const ExtensionUtils = imports.misc.extensionUtils;
 const Me = ExtensionUtils.getCurrentExtension();
 
 // Dev or Prod mode, see Makefile:debug
-var production = true;
+var production = false;
```

Comment:
This has to be defaulting to true

Reply:
Good catch

Reply:
For what it's worth, I kept the developer / logging out of normal users (only folks who don't care about compiling from source) - they just want the extension - in case you are curious where it is used.

Reply:
That makes sense. I don't think it's a huge cuddy to expose a single drop down though. If logging was in the "settings" page, would you be happy to leave it accessible? Maybe developer settings should go in an expander row

Reply:
> That makes sense. I don't think it's a huge cuddy to expose a single drop down though. If logging was in the "settings" page, would you be happy to leave it accessible? Maybe developer settings should go in an expander row

Yes, it should still be accessible on prefs/settings(if this is what you mean), I am OK with an expander, it is just that the expander / section / widget / component would not appear if production == true.